src/Raven.Server/Config/Categories/IndexingConfiguration.cs

```diff
@@ -620,6 +620,13 @@ public virtual bool RunInMemory
         [ConfigurationEntry("Indexing.Corax.VectorSearch.OrderByScoreAutomatically", ConfigurationEntryScope.ServerWideOrPerDatabaseOrPerIndex)]
         public bool CoraxVectorSearchOrderByScoreAutomatically { get; set; }
         
+        [Description("MaxPercentageOfThreadsForEmbeddings")]
+        [DefaultValue(25)]
+        [IndexUpdateType(IndexUpdateType.None)]
+        [ConfigurationEntry("Indexing.Corax.VectorSearch.MaxPercentageOfThreadsForEmbeddings", ConfigurationEntryScope.ServerWideOnly)]
```

Comment:
MaxNumberOfThreadsUsedForEmbeddingGenerationInPercentages

Reply:
Why are we doing percentage instead of direct number?